lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
Correct. We want to keep a heading here (`h3`). The wording should be something general (in case we need to re-use the same heading) but also descriptive. I might also suggest it include the words "form" and "error" so it's clear out of context that there were "errors" in the "form". Does this make sense? @catong maybe you can come up with something clever here?